doc_template/examples_root/examples/connectivity_ex2.py

```diff
@@ -3,9 +3,9 @@
 # tell the cache class what resolution (in microns) of data you want to download
 mcc = MouseConnectivityCache(resolution=25)
 
-# use the ontology class to get the id of the isocortex structure
-ontology = mcc.get_ontology('ontology.csv')
-isocortex = ontology['Isocortex']
+# use the structure tree class to get information about the isocortex structure
+structure_tree = mcc.get_structure_tree()
+isocortex_id = structure_tree.get_structures_by_name('Isocortex')[0]['id']
```

Comment:
I think 'Isocortex' needs to be in a list?

Reply:
Just confirmed that it is supposed to be a list.  Looks like this one works by accident.